### PR TITLE
reducing redundancy

### LINKED ISSUE #2082: [Site] Handbook
**Current Behavior**
The entire handbook uses styling in each page as well as a stylesheet, without the use of common classes.
Also, due to this some pages are not mobile responsive while others are.

**Desired Situation**
Create common classes and a single style sheet for the entire handbook so that it is easier to work with.
Also, add mobile responsivness

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Reducing redundancies , adding mobile responsiveness, removed landing page of handbook, changed font color.
This PR fixes #2082 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 12c6b908ba996e8209d39b566f58888cedb1fc11 at: https://611bfd85d34dde07c5d3c67b--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit aea059cf0934d495ada895f3a885df2f46a551f1 at: https://611c0f003946270b3a7286aa--layer5ng.netlify.app

**Comment by Tanuj22** [comment]
@warunicorn19 what's the url to check the page?

File: src/sections/Community/Handbook/Handbook.style.js [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+
+.highlight{
+
+  font-weight: 600;
+}
+.content{
+  width:100%;
+  padding-bottom: 2rem;
+ul > li {
+    color:#00d3a9;
```
Comment (Tanuj22) [inline src/sections/Community/Handbook/Handbook.style.js:13]: Can you use colors from the theme instead of hardcoding everywhere

**Comment by warunicorn19** [comment]
https://611c0f003946270b3a7286aa--layer5ng.netlify.app/handbook/about

**Comment by Tanuj22** [comment]
Can you make all pages `/community/handbook/.....` and also change the about to just `/community/handbook` till we have a home page

File: src/sections/Community/Handbook/Handbook.style.js (hunk 1 above)
Comment (warunicorn19) [reply src/sections/Community/Handbook/Handbook.style.js:13]: @Tanuj22 the colors which are not in the theme I'm leaving them hardcoded, is that okay?

Comment (Tanuj22) [reply src/sections/Community/Handbook/Handbook.style.js:13]: We shouldn't be using any colors that are not in the theme though :thinking:

Comment (Tanuj22) [reply src/sections/Community/Handbook/Handbook.style.js:13]: Which color is being used that is not in the theme?

Comment (warunicorn19) [reply src/sections/Community/Handbook/Handbook.style.js:13]: These are there from before😅, I havent added any separate colors

Comment (warunicorn19) [reply src/sections/Community/Handbook/Handbook.style.js:13]: #f1f1f1, #929496 these are colors added for hover effect

Comment (warunicorn19) [reply src/sections/Community/Handbook/Handbook.style.js:13]: these can probably be replaces by `lighter gray` which a color in our theme

Comment (Tanuj22) [reply src/sections/Community/Handbook/Handbook.style.js:13]: Okay. I can't locate the colors though. If replacing works go ahead

**Comment by l5io** [comment]
🚀 Preview for commit 23797c922fa1c0418e5ca9bd18a32abeb8a1cd99 at: https://611d10506d69570f93a01b14--layer5ng.netlify.app

**Review by Tanuj22 — APPROVED** [review]
Thank you @warunicorn19

**Pull request merged**